pkg/rancher-desktop/backend/wsl.ts

```diff
@@ -852,7 +852,7 @@ export default class WSLBackend extends events.EventEmitter implements VMBackend
       await this.writeFile(`/etc/moproxy/proxy.ini`, '; no proxy defined');
     }
 
-    await this.modifyConf('moproxy', { MOPROXY_NOPROXY: proxy.noproxy.join(',') });
+    await this.modifyConf('moproxy', { MOPROXY_NOPROXY: proxy.noproxy.filter(ip => ip).join(',') });
```

Comment:
Should this strip whitespace & make things unique? I end up with:

> `MOPROXY_NOPROXY="…,240.0.0.0/4,foo,foo ,foo  ,foo   "`

I'm mostly worried that the user will copy & paste something and then end up with space at the end of the line unintentionally, and then get confused that things aren't working.

Reply:
Yes, obviously. I was just filtering out empty lines, but need to do more.